pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_module-metadata-in-pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+[id="module-metadata-in-pantheon_{context}"]
+= Module metadata in {ProductShortName}
+
+Each module in {ProductShortName} has associated metadata for search engine optimization (SEO). SEO improves the findability of modules.  
+
+There are two types of module-metadata in {ProductShortName}:
```

Comment:
Unnecessary hyphen in "module metadata".

Reply:
Also, should we explicitly say "two types", or should we just preface the list with a generic statement?

```adoc
The types of module metadata in {ProductShortName} are as follows:
```

Reply:
removed hyphen. Agree with not saying two types explicitly, changed as suggested.